freshmaker/parsers/internal/manual_rebuild.py

```diff
@@ -51,10 +52,17 @@ def parse_post_data(self, data):
             errata = Errata()
             advisory = ErrataAdvisory.from_advisory_id(errata, errata_id)
 
-            event = ManualRebuildWithAdvisoryEvent(
-                msg_id, advisory, data.get("container_images", []), data.get("metadata", None),
-                freshmaker_event_id=data.get('freshmaker_event_id'), manual=True,
-                dry_run=dry_run, requester=data.get('requester', None))
+            # If we want to manually retrigger bundle rebuild, create
+            # BotasErrataShippedEvent event
+            if advisory.state == "SHIPPED_LIVE" and \
+                    advisory.reporter.startswith('botas'):
+                event = RetriggeredBundleRebuild(
+                    msg_id, advisory, data.get('freshmaker_event_id'), dry_run)
+            else:
+                event = ManualRebuildWithAdvisoryEvent(
+                    msg_id, advisory, data.get("container_images", []), data.get("metadata", None),
+                    freshmaker_event_id=data.get('freshmaker_event_id'), manual=True,
+                    dry_run=dry_run, requester=data.get('requester', None))
         else:
             event = ManualBundleRebuild(msg_id, data.get('container_images', []),
```

Comment:
`ManualBundleRebuild` now doesn't support advisory id, this should be adjusted, release-driver will also need the support of advisory id in `ManaualBundleRebuild` event.

Reply:
Yes, and it was the part of the implementation, I was pretty sure during the implementation of this event, that there will be no advisory submitted/passed from BOTAS, so BOTAS will submit ONLY bundles + container_images, so I should work with only submitted information.